glogger/extra_adapter.py

```diff
@@ -34,11 +34,11 @@ def process(self, msg: Any, kwargs: MutableMapping[str, Any]) -> Tuple[Any, Muta
                 other_kwargs[k] = v
 
         # Merge other kwargs into extra:
-        extra = logging_kwargs.get("extra", {}) | other_kwargs
+        extra: Mapping[str, object] = {**logging_kwargs.get("extra", {}), **other_kwargs}
```

Comment:
`object`? shouldn't it be Any?

Reply:
Hmm I copied from `extra: Mapping[str, object]` in the c'tor, and later we have `Mapping[str, Any]`, I suppose ALL need to be `Mapping[str, Any]` right?